install_files/udev/99-razer.rules

```diff
@@ -24,8 +24,8 @@ ATTRS{idProduct}=="0501|0504|0506|0510", \
     ATTRS{idVendor}=="1532", \
     ENV{ID_RAZER_CHROMA}="1", ENV{RAZER_DRIVER}="razerkraken"
 
-# Accessories - Mug, Base Station, HDK
-ATTRS{idProduct}=="0f07|0f08|0f09", \
+# Accessories - Mug, Base Station, HDK, Nommo Pro, Nommo Chroma
+ATTRS{idProduct}=="0f07|0f08|0f09|0517|0518", \
```

Comment:
sorting by pid here as well please

Reply:
Reordered by PID, reordered accessory names to match